app/models/solicitation.rb

```diff
@@ -215,13 +216,21 @@ def set_siret_and_region
       .or(where(email: solicitation.email))
   }
 
+  scope :banned, -> {
+    where(banned: true)
+  }
```

Comment:
```suggestion
  scope :banned, -> { where(banned: true) }
```
Je l'aurai mis sur une seule ligne